opentaxii/taxii2/validation.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from opentaxii.taxii2.exceptions import ValidationError
+from stix2 import parse
+from stix2.exceptions import STIXError
+
+
+def validate_envelope(json_data: str, allow_custom: bool = False) -> None:
+    """
+    Validate if ``json_data`` is a valid taxii2 envelope.
+
+    :param str json_data: the data to check
+    :param bool allow_custom: if true, allow non-standard stix types
+    """
+    if not json_data:
+        raise ValidationError("No data.")
```

Comment:
Nit-picking. Usually exception messages are formatted like headings not sentences.